src/lua/color_display_480x272/TOOLS/elrsV2.lua

```diff
@@ -459,6 +463,14 @@ local function refreshNext()
   end
 end
 
+local function lcd_title()
+  lcd.clear()
+  lcd.drawFilledRectangle(0, 0, LCD_W, 30, TITLE_BGCOLOR)
+  local title = (allParamsLoaded == 1 or elrsFlags > 0) and deviceName or "Loading..."
+  lcd.drawText(1, 5, title, MENU_TITLE_COLOR)
```

Comment:
Should just be `lcd.drawText(1, 5, title)` as you are writing in the title line and the colors are automatically used

Reply:
But... why... why does that work?! Updated with your suggestion and it works as expected. Thanks so much!